### PR TITLE
Forgot password

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Created layout forgot password

https://cdn.fromdoppler.com/doppler-ui-library/build162/forgot-password.html

### CODE REVIEW

File: assets/scss/modules/_forms.scss [hunk 1]
```diff
@@ -55,9 +55,16 @@
   }
 }
 
+/* signup form */
+
+.signup-form{
```
Comment (andresmoschini) [inline assets/scss/modules/_forms.scss]:
```suggestion
.signup-form {
```

File: assets/scss/modules/_forms.scss [hunk 2]
```diff
@@ -66,12 +73,36 @@
...
+  margin: $dp-spaces--lv4 $dp-spaces--lv0;
+  font-size: $dp-font-size-form;
+  font-style: italic;
+  font-family: $dp-font-family;
+  position: relative;
+}
+
+/* forgot password form */
+
+.forgot-pass{
```
Comment (andresmoschini) [inline assets/scss/modules/_forms.scss]:
```suggestion
.forgot-pass {
```

File: assets/scss/modules/_forms.scss [hunk 3]
```diff
@@ -66,12 +73,36 @@
...
+  margin-top: $dp-spaces--lv6;
+}
+
+.triangle-right {
+    width: 0;
+    height: 0;
+    padding-left: 17px;
+
+  &:before{
```
Comment (andresmoschini) [inline assets/scss/modules/_forms.scss]:
```suggestion
  &:before {
```

File: assets/scss/modules/_forms.scss [hunk 4]
```diff
@@ -66,12 +73,36 @@
...
+/* forgot password form */
+
+.forgot-pass{
+  margin-top: $dp-spaces--lv6;
+}
+
+.triangle-right {
+    width: 0;
+    height: 0;
+    padding-left: 17px;
```
Comment (andresmoschini) [inline assets/scss/modules/_forms.scss]: Probably we should use a normalized space constant here.

**Comment by andresmoschini** [comment]
👍

**Comment by andresmoschini** [comment]
🚀

**Comment by GusBaamonde** [comment]
all the links in the pages were added. The style of the menu was adjusted.
https://cdn.fromdoppler.com/doppler-ui-library/build163/forgot-password.html

**Comment by andresmoschini** [comment]
Something weird happened:

image

**Comment by GusBaamonde** [comment]
> Something weird happened:

They are the icons that are broken

**Pull request merged**